src/client/components/AdminPanel/BruinCastListings.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState, useEffect } from 'react';
+
+import { View } from '@instructure/ui-view';
+import { Text } from '@instructure/ui-text';
+import { Button } from '@instructure/ui-buttons';
+import { Grid } from '@instructure/ui-grid';
+import { TextInput } from '@instructure/ui-text-input';
+import { ScreenReaderContent } from '@instructure/ui-a11y-content';
+import axios from 'axios';
+
+import { BruinCastListingsToggle } from './BruinCastListingsToggle';
+import { ltikPromise } from '../../services/ltik';
+
+export const BruinCastListings = () => {
+  const [searchTerm, setSearchTerm] = useState('');
+  const [mediaListings, setMediaListings] = useState([]);
+
+  console.log(mediaListings);
```

Comment:
@chern Included console.log again... Maybe we should add an ESlint rule back to prevent this. Can you enable this ESlint rule back in the .eslint.rc file? https://eslint.org/docs/rules/no-console